seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/controller/ScriptController.java

```diff
@@ -120,7 +120,7 @@ public Result<Void> updateScriptParam(@ApiParam(value = "script id", required =
     @GetMapping("/{scriptId}/param")
     @ApiOperation(value = "fetch script param", httpMethod = "GET")
     @ApiImplicitParams({
-            @ApiImplicitParam(name = "id", value = "script id", dataType = "Integer"),
+        @ApiImplicitParam(name = "id", value = "script id", dataType = "Integer"),
```

Comment:
revert?

Reply:
This is to fix the code format problem reported by `checkstyle`.

![image](https://user-images.githubusercontent.com/37063904/217525068-ead6912a-4016-4b2e-8e79-bedfd1b81aba.png)